setup.py

```diff
@@ -47,8 +47,8 @@
     # Additional entries you may want simply uncomment the lines you want and fill in the data
     # url='http://www.my_package.com',  # Website
     install_requires=[
-        'numpy>=1.16.0',
-        'mdanalysis>=2.0.0b0',
+        'numpy>=1.20.0',
+        'mdanalysis>=2.3.0'
```

Comment:
Any reason for being so up to date here? i.e. is there actually an API change that prevents users from using >2.0?

Reply:
Because of NEP29, MC runs with Python 3.8+ and numpy>1.20. Shouldn't I set the version of MDA that matches numpy>1.20 and min Python 3.8? Otherwise, how should I choose it? If I choose MDA==2.1.0, for example, wouldn't it defeat the point of NEP29?

Reply:
I'm not sure I understand, this would be `>=<version number>`. Since 2.0.0+ supports py3.8 and numpy>=1.20, wouldn't this be fine?

Reply:
Thanks for clarifying. I thought I should get the version that supports _min_ numpy>=1.20 and was following the release notes for MDA 2.3.0 that say "the _minimum_ NumPy version to 1.20.0.".  Set to MDA>=2.0.0 .